pkg/secrets-store/nodeserver.go

```diff
@@ -65,6 +65,8 @@ const (
 	// CSIPodServiceAccountTokens is the service account tokens of the pod that the mount is created for
 	CSIPodServiceAccountTokens = "csi.storage.k8s.io/serviceAccount.tokens" //nolint
 
+	SecretStoreVolumeID = "secrets-store-csi-driver.sigs.k8s.io/volume.id"
```

Comment:
Should have a test to prove this is present?

Reply:
I can add a test and doc note about it

Reply:
Turns out there is already a test ensuring this is present. I'll just add teh docs